dti-website-server/src/server.ts

```diff
@@ -0,0 +1,15 @@
+import express, { RequestHandler, Request, Response } from 'express';
+import serverless from 'serverless-http';
+
+// Constants and configurations
+const app = express();
+const router = express.Router();
+
+router.get('/', (req: Request, res: Response) => {
+  console.log('Ping!');
+  res.send('Pong!');
+});
```

Comment:
I don't understand why we need this server file to collect analytics. If we need analytics, we can always do it on client side using google analytics.

Reply:
This is just a placeholder for something I wanted to try out, so it's not really for analytics, but I've just realized I can manually add the workflow to main, which is why I opened the PR in the first place. but I see the PR review is how you're reviewing the member JSON changes, so I'm going to switch up the workflow to only be on merges to master. Then I'll add a button in IDOL that let's admins redeploy the website and pull the data from IDOL at will, which would allow us to update it immediately if we so choose. I'm closing this PR for now, but I'll make those changes now that I understand the structure a bit better.